drivers/include/net/netdev2.h

```diff
@@ -10,15 +10,153 @@
  */
 
 /**
- * @defgroup    drivers_netdev_netdev2 Generic Low-Level Network Driver Interface v2
+ * @defgroup    drivers_netdev_netdev2 Network Device Driver Interface
  * @ingroup     drivers_netdev
  * @brief       This is the second version of a generic low-level network
  *              driver interface
  * @{
  *
- * This interface is supposed to be a low-level interface for network drivers.
+ * @section sec_about About
```

Comment:
Mh... after reading this document I must say: you are mixing styles yourself: you are using e.g. markdown formatting for **bold** below instead of the Doxygen-style: `@b bold`